docs/topics/db/optimization.txt

```diff
@@ -373,3 +413,8 @@ This also applies to :class:`ManyToManyFields
     my_band.members.add(my_friend)
 
 ...where ``Bands`` and ``Artists`` have a many-to-many relationship.
+
+List unpacking also works to add querysets or lists of objects::
+
+    new_band_members = [me, my_friend]
+    myband.members.add(*new_band_members)
```

Comment:
I don't see much value in these change and it's definitely not related with this ticket. Please remove it.